src/tests/Input.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import Input from '../components/Input';
+import data from './__mocks__/apimock';
+
+describe('Input page renders as required', () => {
+  it('Should be rendering 27 regions', () => {
+    render(<BrowserRouter><Input data={data} /></BrowserRouter>);
+
+    expect(data.length).toBe(27);
+  });
+
+  it('Should be rendering 27 regions', () => {
```

Comment:
- [OPTIONAL] These two **`test examples`** have the same **`test titles`** and they all seem to be testing for relatively the same outcome. Consider doing away with one of them to clean up your tests.